plugins/source/gitlab/resources/services/groups/groups_fetch.go

```diff
@@ -0,0 +1,41 @@
+package groups
```

Comment:
can you gen it? it's a common pattern here

Reply:
I agree that there are many opportunities to incorporate more/better codegen. But I don't want to invest in the implementation until we are sure we have the structure correct or users are requesting more and more resources